docs/compilers/CSharp/Compiler Breaking Changes - DotNet 9.md

```diff
@@ -0,0 +1,22 @@
+# This document lists known breaking changes in Roslyn after .NET 8 all the way to .NET 9.
+
+## .editorconfig values no longer support trailing comments
+
+***Introduced in Visual Studio 2022 version [TBD]***
+
+The compiler is updated based on the EditorConfig specification clarification in
+[editorconfig/specification#31](https://github.com/editorconfig/specification/pull/31). This changes the way values are
+passed to analyzers for lines with the following form:
```

Comment:
The examples are great, but this might be clear too if we just say "comments now must be on their own line; comments after a value are treated as a part of the value."

Reply:
➡️ Updated the text